README.md

```diff
@@ -3,7 +3,7 @@
 
 **Qiskit** is an open-source framework for working with noisy quantum computers at the level of pulses, circuits, and algorithms.
 
-Qiskit is made up of elements that work together to enable quantum computing. This element is **Terra** and is the foundation on which the rest of Qiskit is built.
+The core component of Qiskit is `qiskit-terra`.
```

Comment:
Do we want to add a sentence saying that e.g. the applications and other packages are built on it, or just leave as is?

Reply:
I would prefer leaving it as it is for now.